### ISSUE TITLE
Add new layer for 2.8.8 changes

### ISSUE THREAD

**Issue by MarkehMe** [MarkehMe/FactionsFramework#5]
As reported here:

https://github.com/MarkehMe/FactionsWarps/issues/6

FactionsFramework requires a new layer to support changes in 2.8.8.

**Comment by MarkehMe** [comment]
Fixed in ff15b8cb4cd9042932663c4438b0bf543de0b2e9

**Comment by MarkehMe** [comment]
🎉  **FactionsFramework v1.1.1 has been released and addresses this issue** 🎉 

You can grab it from:
* [GitHub](https://github.com/MarkehMe/FactionsFramework/releases/tag/1.1.1)
* [Spigot Resources](https://www.spigotmc.org/resources/factions-framework.22278/update?update=89723)
* [BukkitDev](http://dev.bukkit.org/bukkit-plugins/factionsframework/files/3-factions-framework-v1-1-1/)

```
v1.1.1 Changelog:

* Added a missing 1.8 Layer
* Added new 2_8_8 layer support Factions 2.8.8 and newer versions
* Minor 2_8_6 layer fixes
```
